build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    java
+    id("org.jetbrains.intellij") version "0.7.2"
+}
+
+intellij {
+    type = "IC"
+    version = "2021.1"
+    setPlugins("java")
+}
+
+group = "org.jetbrains.research.anticopypaster"
+version = "3.3"
```

Comment:
I suggest using the following version naming convention:
'A-B'
A - version of IDE (for example 2021.3)
B - version of the plugin (2.0)

Reply:
changed